src/Views/FormPages/Addresses.jsx

```diff
@@ -23,6 +24,7 @@ function Addresses(props){
     const homeAddressSectionField = fields.find(item => item.uuid === "63552bb6-6afb-46e1-8148-860242917a22");
     const streetAddressField = fields.find(item => item.uuid === "6dcb9e8c-b40a-4cda-ba5c-06b98c3375f4");
     const cityField = fields.find(item => item.uuid === "7e39a528-7518-40cb-b7b6-b635864dc117");
```

Comment:
cityField and stateField can be removed here. These are being used within their field components.